furusiya/main.py

```diff
@@ -46,4 +44,8 @@ def main_menu():
         elif choice == 2:  # quit
             break
 
+if config.has("seed"):
+    seed = config.get("seed")
+    random.seed(seed)
+    print("Seeding as universe #{}".format(seed))

```

Comment:
If `seed` is set to null, this statement would print `Seeding as universe #None`. Perhaps in this case it should generate a random seed, seed the game, and then print it?

Reply:
The print statement is not that important. By default, `None` means `random` calls use a random seed. (I tested it.)